mhkit/tests/utils/test_cache.py

```diff
@@ -132,7 +146,11 @@ def test_handle_caching_cdip_file_extension(self):
         - The cache file with a ".pkl" extension is successfully created at the expected file path.
         """
         cache_dir = os.path.join(self.cache_dir, "cdip")
-        handle_caching(self.hash_params, cache_dir, data=self.data)
+        handle_caching(
+            self.hash_params,
+            cache_dir,
+            cache_content={"data": self.data, "metadata": None, "write_json": None},
+        )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args